make/Images.gmk

```diff
@@ -85,10 +86,22 @@ JLINK_TOOL := $(JLINK) -J-Djlink.debug=true \
 
 JLINK_JRE_EXTRA_OPTS := --no-man-pages --no-header-files --strip-debug
 
+ifeq ($(JLINK_PRODUCE_RUNTIME_LINK_JDK), true)
+  JDK_LINK_OUTPUT_DIR := $(RL_INTERMEDIATE_IMAGE_DIR)
+else
+  JDK_LINK_OUTPUT_DIR := $(JDK_IMAGE_DIR)
+endif
+
 ifeq ($(JLINK_KEEP_PACKAGED_MODULES), true)
-  JLINK_JDK_EXTRA_OPTS := --keep-packaged-modules $(JDK_IMAGE_DIR)/jmods
+  ifeq ($(JLINK_PRODUCE_RUNTIME_LINK_JDK), true)
```

Comment:
I don't get it. Why don't you use the  JDK_LINK_OUTPUT_DIR from just above?

Reply:
Makes sense. I can fold both `JLINK_JDK_EXTRA_OPTS` into one. `JLINK_JDK_EXTRA_OPTS := --keep-packaged-modules $(JDK_LINK_OUTPUT_DIR)/jmods`

Reply:
Fixed now.